pandas/tests/indexes/numeric/test_numeric.py

```diff
@@ -531,7 +535,10 @@ def test_constructor(self, dtype):
         res = Index([1, 2 ** 63 + 1], dtype=dtype)
         tm.assert_index_equal(res, idx)
 
-    @pytest.mark.xfail(reason="https://github.com/numpy/numpy/issues/19146")
+    @pytest.mark.xfail(
+        not (is_platform_arm and is_platform_mac()),
+        reason="https://github.com/numpy/numpy/issues/19146",
+    )
```

Comment:
this is now failing for me locally on non-ARM mac

Reply:
failing on ci too.